public/leie.html

```diff
@@ -77,13 +77,14 @@ <h2>Komplette utlånsvilkår</h2>
           </li>
           <li>Leiepriser
             <ol>
-              <li>Personell faktureres med kr 273,- per time per funksjonær.</li>
+              <li>Personell faktureres med kr 407,- per time for skjenkemester og kr 325,- per time per funksjonær.</li>
               <li>
                 I tillegg til arrangementets varighet betales alt personell én time ekstra for forberedelser og
                 rydding.
               </li>
               <li>For hvert utlån tilkommer det normalt et slitasjegebyr på kr 500.</li>
               <li>Alle priser inkluderer 25% merverdiavgift.</li>
+              <li>Hvis utlånet m/rydding går utover siste ordinære avgang for kollektivtransport, vil det faktureres gebyr til taxi på maks 800 per personell. </li>
```

Comment:
Dette punktet bør kanskje stå over dette?

Det er forøvrig litt interessant rundt taxi. Når vi kostnadsfører dette så har vi fradrag for lav mva, men jeg er litt usikker hvordan det blir når vi fakturerer det videre (hvorvidt det da blir høy mva eller om vi kan fakturere det med lav sats).

Reply:
@henrist ser ut som at taxi faktureres med høy mva ref. https://www.jithomassen.no/viderefakturering-av-utlegg/

Reply:
Fin avklaring!

800 med 25 % MVA = 640 eks mva
12 % på dette blir kr 716,8 inkl mva (12 %)

Dette er altså det øverste beløpet en taxi-regning kan koste hvor vi får igjen alle pengene ved å fakturere videre.